sold.cc

```diff
@@ -821,6 +822,17 @@ class Sold {
         }
     }
 
+    void ConstructFilenameToSoname() {
+        for (const auto& l : link_binaries_) {
```

Comment:
I think it's better to check if `soname` in `link_binaries_` are unique.

Reply:
I do it in the future PR when I change the order of loading.